aptos-move/framework/aptos-framework/sources/transaction_context.move

```diff
@@ -1,4 +1,7 @@
 module aptos_framework::transaction_context {
+    /// Return a globally unique identifier 
+    public native fun create_guid(): address;
```

Comment:
Or provide `get_tx_hash()` and `guid_counter()` native functions, and implement the `create_guid()` in Move

Reply:
If we go that route, then we can explore multiple solutions and leave it more to the user to dictate their preferred approach.

Reply:
Isn't it generally preferable to provide as simple interface as possible, and reduce the cognitive load on the smart contract developer? In terms of functionality, the developer would achieve the same with both the options, but the developer has to put more effort into how to construct GUIDs if we expose `get_tx_hash` and `guid_counter` without an additional benefit.

Reply:
The `get_tx_hash` is helpful in many scenarios, `guid_counter` may be just for the `create_guid`. And I think the `new_table_handle` is repeated with `create_guid`; if we do not reuse the `guid_counter`, it will produce the repeat `table_handle` and `guid`.

Reply:
@jolestar - more details on the choice : https://www.notion.so/aptoslabs/Move-Parallelism-for-MetaPixel-982afa7c10ca494fad235fd1aa4318f4?pvs=4#d87bb42cc7a648d39fa3e699a4fffd35

guid_counter is scoped separately, while table_handle isn't